package com.company.chapter12errorhandlingandexceptions.tasks.task4;

public class SecondException extends Exception {
}
